File: global/php/logic/utils/classes/Task.php
<?php

class Task{
	private string $nome;

	private bool $concluida;

	public function __construct($nome) {
        $this->nome = $nome;
        $this->concluida = false;
    }

	public function getNome(): string
	{
		return $this->nome;
	}

	public function setNome($nome): void
	{
		$this->nome = $nome;
	}

	public function getConcluida(): bool
	{
		return $this->concluida;
	}

	public function setConcluida($concluida): void
	{
		$this->concluida;
	}
}

File: backend/dotnet/templates/Task.cs
namespace ToDoList.Templates
{
    public class Task
    {
        public string nome;

        public bool concluida;

        public Task(string nome)
        {
            this.nome = nome;
            concluida = false;
        }

        public string Nome
        {
            get { return nome; }
            set { nome = value; }
        }

        public bool Concluida
        {
            get { return concluida; }
            set { concluida = value; }
        }
    }
}